### ISSUE TITLE
Launcher Problems

### ISSUE THREAD

**Issue by bluetoot420** [AOF-Dev/MCinaBox#478]
Why is it when i set up everything needed for my launcher and i try to use the kauncher it just crashes the app then goes back to home screen or wherever i was before tabbing into mcinabox. Pls help me with this, and yes my ohone might be a piece of shit but it can still handle about 2gb right now so pls anyone help me

**Comment by Kotarou2k4** [comment]
Which version ?

**Comment by bluetoot420** [comment]
> Which version ?

I tried 3 versions:
Mc 1.12.2
Mc 1.8.9
Mc 1.16.3

**Comment by Kotarou2k4** [comment]
Maybe you allocated too much ram

**Comment by Kotarou2k4** [comment]
Do it show any bug on the to right screen ?

**Comment by bluetoot420** [comment]
> Maybe you allocated too much ram 

1.16 i put 500 then back to 256
1.8.9 same thing
1.12 1500 to 500 back to 256

**Comment by bluetoot420** [comment]
> Do it show any bug on the to right screen ?

It says the keyboard thing but it disappears after time cuz my keyboard is loading

**Comment by Kotarou2k4** [comment]
So everything still working normal but you enter the game, you should ask longjunyu to fix this

**Comment by bluetoot420** [comment]
> So everything still working normal but you enter the game, you should ask longjunyu to fix this

Yes.. i press the launch of any version i have wich is currently just:
1.8.9 forge
1.8.9
1.12.2
1.16.3 
And it doesnt work it just tabs me out and crashes the launcher

**Comment by CosineMath** [comment]
You should have a look on log.
